it/google-cloud-platform/src/test/java/com/google/cloud/teleport/it/gcp/dataflow/FlexTemplateClientTest.java

```diff
@@ -151,7 +152,7 @@ public void testLaunchNewJob() throws IOException {
             .setRunner("Dataflow")
             .setParameters(ImmutableMap.of(PARAM_KEY, PARAM_VALUE))
             .build();
-    assertThat(actual).isEqualTo(expected);
+    Truth.assertThat(actual).isEqualTo(expected);
```

Comment:
Static imports is often a good practice for this kind of assertion. Any reasoning for qualifying it?

Reply:
These changes were not intended. Fixed.